### ISSUE TITLE
react-js / import error on @aws-amplify/ui-react/styles.css

### ISSUE THREAD

**Issue by NickRHR** [aws-amplify/amplify-ui#1061]
Dear all,

I'm currently facing the following error when importing @aws-amplify/ui-react/styles.css in my react-js project:
`import '@aws-amplify/ui-react/styles.css';`

Error:
./node_modules/@aws-amplify/ui-react/dist/index.css (./node_modules/css-loader??ref--6-oneOf-3-1!./node_modules/postcss-loader/src??postcss!./node_modules/@aws-amplify/ui-react/dist/index.css)
Module not found: Can't resolve './@aws-amplify/ui/styles.css' in '[PROJECTNAME]/node_modules/@aws-amplify/ui-react/dist'

Dependencies:
    "@aws-amplify/ui-react": "^2.1.5",
    "@testing-library/jest-dom": "^5.16.1",
    "@testing-library/react": "^12.1.2",
    "@testing-library/user-event": "^13.5.0",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-scripts": "^2.1.8",
    "web-vitals": "^2.1.2"

Thanks in advance for your help,

Regards

NTC

**Comment by reesscot** [comment]
Hi @NickRHR,
What build system are you using? This sounds similar to an issue other customers were reporting:
https://github.com/aws-amplify/amplify-ui/issues/1020

This should be [fixed](https://github.com/aws-amplify/amplify-ui/pull/1024) in the next release. If you'd like to verify the [fix](https://github.com/aws-amplify/amplify-ui/pull/1024) in your application, follow these [instructions](https://github.com/aws-amplify/amplify-ui/issues/1020#issuecomment-998361143).

**Comment by NickRHR** [comment]
Hi @reesscot and thanks a lot for your prompt follow-up & sorry I did not find about it myself,
I'm building through react-scripts (created my app via CRA) -
Tested the fix you mentioned using the instructions you provided, it worked !
Thanks a lot
NTC

**Comment by khiz-k** [comment]
Hi, 

I'm having this same issue on nextjs and tried the fix in the comment, which was to update my @aws-amplify/ui-react version to that pre-release but it didn't work.

I deleted my node_modules, package-lock.json and yarn.lock.  Then I ran yarn and yarn dev.  I even tried that whole process with npm and npm run dev.  Nothing worked.

My error message is the following:
Module not found: Can't resolve '@aws-amplify/ui-react/styles.css'
  19 | 
> 20 | import "@aws-amplify/ui-react/styles.css";
  21 |
